src/server/pfs/fuse/options.go

```diff
@@ -82,17 +86,17 @@ func (o *Options) validate(c *client.APIClient) error {
 	if o == nil {
 		return nil
 	}
-	for repo, opts := range o.RepoOptions {
+	for _, opts := range o.RepoOptions {
 		if opts.Write {
 			if uuid.IsUUIDWithoutDashes(opts.Branch) {
-				return errors.Errorf("can't mount commit %s@%s in Write mode (mount a branch instead)", repo, opts.Branch)
+				return errors.Errorf("can't mount commit %s@%s %s in Write mode (mount a branch instead)", opts.Repo, opts.Branch)
```

Comment:
I think there might be a printf bug here (there are three `%s` tokens in the format string, but only two arguments: `opts.Repo` and `opts.Branch`). I think you meant to add the name as a printf argument but never quite got to it.

Reply:
good catch, I also forgot to thread the mount name through into the RepoOpts struct, which I've done now